jobs/pgbackup/templates/bin/pgbackup

```diff
@@ -62,13 +118,17 @@ backup_databases() {
 	else
 		echo "FAILED to back up!"
 	fi
+	rm -f "/tmp/backup.lock"
 }
 
 echo "pgbackup starting up"
-# FIXME: acquire a lock to ensure we aren't running simultaneous dumps
 
-backup_databases
+# if a backup.lock file exist then we assume that an other dump is running, so we will retry to backup in the next run
+if [ ! -f "/tmp/backup.lock" ]; then
+    backup_databases
+else
+    echo "ERROR : an other dump is running or the last backup failed , see logs for more details"
```

Comment:
lock is released when backup fails, should read "an other dump is still running skipping this backup, see logs for more details"